config.go

```diff
@@ -948,6 +958,12 @@ func loadConfig() (*config, []string, error) {
 		dcrdLog.Warnf("%v", configFileError)
 	}
 
+	// Warn if old testnet directory is present.
```

Comment:
I would suggest moving this before the previous block since, as its comment call out, it "should go directly before the return".